### ISSUE TITLE
Flash APs with firmware from nodewatcher

### ISSUE THREAD

**Issue by mitar** [cloyne/network#84]

**Comment by ecawthon** [comment]
It looks like [recent versions of UniFi block custom firmware](https://openwrt.org/toh/ubiquiti/unifi): 

> NOTE for UAP-v2: As for July 2018, Ubiquiti seems to have blocked custom firmware installation possibility by using signatures on all 3.7 and later brand firmwares. As long as there is no brand pre-3.7 firmware available (in order to downgrade) for UAP-v2 devices, installing OpenWrt/LEDE could be impossible, as seen on the forum https://forum.lede-project.org/t/are-ubiquiti-actively-blocking-installation-of-third-party-firmware/4301 and https://forum.lede-project.org/t/are-ubiquiti-blocking-lede-installation/3877. If you want liberty, do not buy Ubiquiti. 

I'm unsure of what if anything this means for our existing hardware. @mitar do you have advice on what direction(s) to pursue in light of this?

**Comment by mitar** [comment]
So I generally stopped installing OpenWrt/LEDE on Ubuiqiti devices anyway. I use original firmware and then have other devices around which use OpenWrt/LEDE. Because proprietary drivers are simply better. So I leave Ubuiqiti as dump bridged devices (so no routing) and put a router with OpenWrt/LEDE behind it (for example with mesh routing protocol).

But generally it also means that if you are using original firmware, maybe there is a reason not to just automatically upgrade always to latest but leave it on old. So that you can move to OpenWrt/LEDE if you want. But in this particular story, I prefer performance over freedom.

For APs in Cloyne there is already OpenWrt there, so we should not worry about that.

If you are buying any new APs you might want to decide what to do.

**Comment by mitar** [comment]
Also, this comment is unrelated with this issue. So APs already have OpenWrt so it is easy to flash with other OpenWrt image.

**Comment by clonm** [comment]
@mitar thanks for the response. I had commented on this issue because I assumed nodewatcher was contingent on our use of OpenWRT - is that the case or can/should we use nodewatcher without?

> I leave Ubiquiti as dump bridged devices (so no routing) and put a router with OpenWrt/LEDE behind it

I'm not sure I understand what you're saying here - are you describing something in the current Cloyne setup? sorry for the basic questions - trying to piece it together...

**Comment by mitar** [comment]
> I assumed nodewatcher was contingent on our use of OpenWRT - is that the case or can/should we use nodewatcher without?

Yes, but you run OpenWrt on router devices, not antennas, which you leave in bridge mode with original firmware.

> I'm not sure I understand what you're saying here - are you describing something in the current Cloyne setup? sorry for the basic questions - trying to piece it together...

Yes, in current Cloyne setup is like that as well. All antennas you have are with original firmware in bridge mode. Routing you do on other devices.
